project/ProjectSettings.scala

```diff
@@ -32,12 +32,12 @@ object ProjectSettings {
     Compile / packageDoc / publishArtifact := false,
     Compile / doc / sources := Seq.empty,
     Compile / doc := target.map(_ / "none").value,
-    scalaVersion := "2.12.13",
+    scalaVersion := "2.12.15",
     initialize := {
       val _ = initialize.value
       assert(
-        sys.props("java.specification.version") == "11",
-        "Java 11 is required for this project.",
+        sys.props("java.specification.version") == "17",
+        "Java 17 is required for this project.",
```

Comment:
I'm a little surprised it was necessary to go to Java 17, as I'm doing ok running Ophan on a Apple M1 Mac with Java 11, but apparently [Java 17 is totally fine for Scala](https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html#version-compatibility-table), yay!

Incidentally, I'm running Azul Zula, installed with https://sdkman.io/:

```
java -version
openjdk version "11.0.13" 2021-10-19 LTS
OpenJDK Runtime Environment Zulu11.52+13-CA (build 11.0.13+8-LTS)
OpenJDK 64-Bit Server VM Zulu11.52+13-CA (build 11.0.13+8-LTS, mixed mode)
```